packages/scss/demo/templates/fullmenu.html

```diff
@@ -11,8 +11,14 @@
 	</head>
 	<body>
 		<main class="main">
-			<aside class="navSide">
+			<aside class="navSide" id="navSide">	
 				<nav class="navSide-scrollWrapper">
+					<div class="navSide-item navSide-mobile-toggle" onclick="w3.toggleClass('#navSide', 'is-open')">
```

Comment:
Should have only one class, with only one child (navSide-toggle or navSide-mobileToggle for example)